src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  margin-bottom: 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  overflow: hidden;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
```

Comment:
styles for `nav` bem block should be in separate scss file

Reply:
According to the task, we shouldn't modify the header